source/_posts/2017-08-29-gsoc-2017-support-to-import-export-of-more-formats.markdown

```diff
@@ -0,0 +1,274 @@
+---
+layout: post
+title: "GSoC 2017 : Support to Import & Export of more formats"
+date: 2017-08-29 17:16
+comments: true
+author: Athitya Kumar
+categories: [GSOC 2017, GSOC, Daru, Daru-IO, Import, Export]
+---
+
+## Introduction
+
+> "Hello friend. Hello friend? That's lame." - S01E01 (Pilot), Mr.Robot
+
+My name is Athitya Kumar, and I'm a 4th year undergrad from IIT Kharagpur, India. I
+was selected as a GSoC 2017 student developer by Ruby Science Foundation for project daru-io.
+
+Daru-IO is a plugin-gem to daru gem, that extends support for many Import and Export methods
+of `Daru::DataFrame`. This gem is intended to help Rubyists who are into Data Analysis
+or Web Development, by serving as a general purpose conversion library.
```

Comment:
I believe it OK to linkify "daru-io" and "daru" in the very first paragraphs. Blog post has a lot of relevant links already, but more context was never bad ;)